### ISSUE TITLE
400 Error on Viewing CF "Add Mod" popup, no mods displayed

### ISSUE THREAD

**Issue by ceruleancerise** [gorilla-devs/GDLauncher#976]
**Describe the bug**
When trying to view the CF mod viewer, nothing is displayed. 400 error is obtained from CF server.

**To Reproduce**
1. Right click any instance and click "Manage".
2. Click on the "Mods" tab on the left of the popup.
3. Click on "Add Mod" on the top of the popup.
4. Observe.

**Expected behavior**
Mods from CF are displayed on the instance manager popup.
**Actual behavior**
Nothing is displayed. Please refer to screenshot

**Screenshots**
![image](https://user-images.githubusercontent.com/15795562/124307147-3f26b280-db1c-11eb-8a12-78ec697ff1c5.png)

**Operating System:**
 - OS: Windows 10 20H2

**Additional context**
Currently installed mods can have their CF info viewable, and can also be updated. 
![image](https://user-images.githubusercontent.com/15795562/124307966-68940e00-db1d-11eb-862e-e9b69af340e8.png)

**Comment by RemagOfficial** [comment]
1.17 mods can't be downloaded in launcher because forge is in beta or unreleased, this happens every time a new update is released as the curseforge API is the issue not gdlauncher

**Comment by thedocruby** [comment]
@RemagOfficial so are you saying that the CurseForge API has been updated and GDLauncher hasn't yet? You said "forge is in beta or unreleased", at first i thought you were referring to Forge 1.17, which i realized is not applicable because the problem occurs on Fabric and as I have found it also occurs on 1.16 versions. Also after opening the "Add mods" menu, if you do anything in the menu after that the launcher crashes. I'd say this is a serious problem.
